ethereum/contractBackend_test.go

```diff
@@ -0,0 +1,50 @@
+package ethereum_test
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	ethereumtest "github.com/direct-state-transfer/dst-go/ethereum/test"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/direct-state-transfer/dst-go/client"
+	"github.com/direct-state-transfer/dst-go/ethereum"
+)
+
+func Test_ContractBackend_Interface(t *testing.T) {
+	require.Implements(t, (*client.ContractBackend)(nil), new(ethereum.ContractBackend))
```

Comment:
:laughing: i never saw this yet. Can you not just do it compile-time with  
`var _ client.ContractBackend = &ethereum.ContractBackend{}`?

Reply:
Yes, it also possible that way and is one the idiomatic ways of doing it.

Yet, i find this is a better approach, because the purpose is to test interface satisfaction and hence it fits better into  `_test.go`. Moreover, by moving it to test, the actual package need not import the `dst-go/client`. (Which it will do otherwise just for checking interface satisfaction).